keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -150,8 +150,8 @@ class TitusClusterHandler(
           }
 
           val description = when (version) {
-            null -> "Resizing server group ${desired.moniker} in ${desired.location.account}/${desired.location.region}"
-            else -> "Upserting server group ${desired.moniker} to $version in ${desired.location.account}/${desired.location.region}"
+            null -> "Resize server group ${desired.moniker} in ${desired.location.account}/${desired.location.region}"
```

Comment:
should this be "Resizing" and "Deploying"?

Reply:
Please review https://github.com/spinnaker/keel/pull/1321 instead -- I'll drop this commit from here.

Reply:
Discussed in slack